pypsa/statistics.py

```diff
@@ -538,16 +538,17 @@ def optimal_capacity(
         comps=None,
         aggregate_groups="sum",
         groupby=None,
-        at_port=False,
+        at_port=None,
```

Comment:
I changed this, because None is also leading to the same result as False. However, if someone deliberately wants to set at_port to False, even if it at first sight makes no sense when bus_carrier is set, then at_port=False is still kept and not replaces by True.